specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview/managedCassandra.json

```diff
@@ -0,0 +1,2631 @@
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraClusterStart": {
+            "$ref": "./examples/CosmosDBManagedCassandraClusterStart.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Enqueued Operation to start the cluster."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/status": {
+      "get": {
+        "operationId": "CassandraClusters_Status",
+        "description": "Gets the CPU, memory, and disk usage statistics for each Cassandra node in a cluster.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraStatus": {
+            "$ref": "./examples/CosmosDBManagedCassandraStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully fetched the usage statistics of all nodes in the cluster.",
+            "schema": {
+              "$ref": "#/definitions/CassandraClusterPublicStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/repairStatus": {
```

Comment:
can this simply be a repair action instead of repairStatus?

Reply:
This api, gets status of all repair in the cluster, hence we are using "repairStatus"